<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QListWidget" name="typeField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QListWidget" name="sizeField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>111</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Time of creation</string>
    </property>
   </widget>
   <widget class="QListWidget" name="timeField">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>201</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove file</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
